enos/modules/read_license/main.tf

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+variable "file_name" {}
+
+output "license" {
+  value = file(var.file_name)
```

Comment:
How does this behave if the file_name is invalid/non-existent? It seems like the runs on this PR worked fine, but I was a little surprised by that.

Reply:
it bails out quickly when planning the scenario:
```Error: Invalid function argument

  on ../../modules/read_license/main.tf line 4, in output "license":
   4:   value = file(var.file_name)
    ├────────────────
    │ while calling file(path)
    │ var.file_name is "/tmp/fakenotafile.hclic"
```

Reply:
So this needs to be set in oss? EDIT: Just kidding. I missed the `skip_step` in the read license step.